crates/rooch/src/commands/transaction/commands/get_tx_by_hash_and_index.rs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) RoochNetwork
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::cli_types::{CommandAction, WalletContextOptions};
+use async_trait::async_trait;
+use clap::Parser;
+use moveos_types::h256::H256;
+use rooch_rpc_api::jsonrpc_types::eth::Transaction;
+use rooch_types::error::RoochResult;
+
+/// Get transaction by hash for Ethereum blockchain
+#[derive(Debug, Parser)]
+pub struct GetByHashAndIndexCommand {
+    /// Transaction's hash
+    #[clap(long)]
+    pub hash: H256,
+
+    /// Transaction's index
+    #[clap(long)]
+    pub index: u64,
```

Comment:
`GetByHashAndIndexCommand ` has been removed in earlier versions.
This PR is far away from the main branch and needs to be rebased first for review.

Reply:
It's already in a rebased version, but I forgot removing them.

Please list all the possible removals. Sorry for the long version gap.

Reply:
I've removed `GetByHashAndIndexCommand`.